server/devdocs/index.js

```diff
@@ -196,5 +197,20 @@ module.exports = function() {
 		response.send( marked( fileContents ) );
 	} );
 
+	// return json for the components usage stats
+	app.get( '/devdocs/service/usage-stats', function( request, response ) {
+		var counts = Object.keys( usageStats )
+			.filter( function( moduleName ) {
+				return moduleName.indexOf( 'components/' ) === 0 &&
```

Comment:
Do you mean the work (filter and reduce) on the `usageStats` object?
I am doing it here to return a smaller response, but I could move this logic to a reducer or in `client/devdocs/service.js`